charts/fluentd-elasticsearch/README.md

```diff
@@ -42,90 +42,91 @@ The command removes all the Kubernetes components associated with the chart and
+| `annotations`                                        | Optional daemonset annotations                                                 | `NULL`                                 |
+| `podAnnotations`                                     | Optional daemonset's pods annotations                                          | `NULL`                                 |
+| `configMaps.useDefaults.systemConf`                  | Use default system.conf                                                        | true                                   |
+| `configMaps.useDefaults.containersInputConf`         | Use default containers.input.conf                                              | true                                   |
+| `configMaps.useDefaults.systemInputConf`             | Use default system.input.conf                                                  | true                                   |
+| `configMaps.useDefaults.forwardInputConf`            | Use default forward.input.conf                                                 | true                                   |
+| `configMaps.useDefaults.monitoringConf`              | Use default monitoring.conf                                                    | true                                   |
+| `configMaps.useDefaults.outputConf`                  | Use default output.conf                                                        | true                                   |
+| `extraConfigMaps`                                    | Add additional Configmap or overwrite disabled default                         | `{}`                                   |
+| `awsSigningSidecar.enabled`                          | Enable AWS request signing sidecar                                             | `false`                                |
+| `awsSigningSidecar.resources`                        | AWS Sidecar resources                                                          | `{}`                                   |
+| `awsSigningSidecar.network.port`                     | AWS Sidecar exposure port                                                      | `8080`                                 |
+| `awsSigningSidecar.network.address`                  | AWS Sidecar listen address                                                     | `localhost`                            |
+| `awsSigningSidecar.network.remoteReadTimeoutSeconds` | AWS Sidecar socket read timeout when talking to ElasticSearch                  | `15`                                   |
+| `awsSigningSidecar.image.repository`                 | AWS signing sidecar repository image                                           | `abutaha/aws-es-proxy`                 |
+| `awsSigningSidecar.image.tag`                        | AWS signing sidecar repository tag                                             | `0.9`                                  |
```

Comment:
image tag is 1.0

Reply:
beside... why this big change?
remoteReadTimeoutSeconds is also missing.

Reply:
> beside... why this big change?

If you ignore the whitespace in a side-by-side diff, you'll see that it's just caused by re-indenting the table to align the columns because `awsSigningSidecar.network.remoteReadTimeoutSeconds` has been added and is longer that the other current variable paths.

> remoteReadTimeoutSeconds is also missing.

Sorry, what do you mean here?

Reply:
> image tag is 1.0

Ack, will fix - thx

Reply:
> > remoteReadTimeoutSeconds is also missing.
> 
> Sorry, what do you mean here?

Hmm... maybe i've missed it out before.
now its there.